docs/unified/novacustom/camera-switch.md

```diff
@@ -0,0 +1,9 @@
+# Camera switch setting
+
+Novacustom laptops have the option to disable the webcam completely. After
+disabling this option, the power to the camera is completely cut off.
+
```

Comment:
This section could use a photo or a screenshot of the feature in the setup menu

Reply:
added in 42563eee50bae817d0b20418e1b73d34775acafc